pynecone/constants.py

```diff
@@ -136,7 +136,7 @@
 # The gitignore file.
 GITIGNORE_FILE = ".gitignore"
 # Files to gitignore.
-DEFAULT_GITIGNORE = {WEB_DIR, DB_NAME}
+DEFAULT_GITIGNORE = {WEB_DIR, DB_NAME, "__pycache__/", "*.py[cod]"}
```

Comment:
we should probably add these to the .gitignore file instead

Reply:
What .gitignore file is?

Reply:
In the view of the application developers,  they use high-level to create the basic project by using ```pc init```. Because the .gitignore is contained in the basic project,  they often assume this .gitignore is enough to prevent uploading the building things to the git server.

For the python project, .gitignore often contain  "__pycache__/", "*.py[cod]". 
And pynecone's web application is a python project too. 

__pycache__/, .pyc, .pyo, .pyd are all compiled byte code
Python programmers usually ignore bytecode.

Reply:
ok

Reply:
> we should probably add these to the .gitignore file instead

The .gitignore file in pinecone is distinct from the .gitignore that is created by pc using these constants. This seems like the right approach, and a change that seems like it would benefit everyone, @ElijahAhianyo

Reply:
Or we may need some reason why Pynecone project needs to keep `__pycache__` on git. 
If there is some reason to keep `__pycache__` on pynecone project, then we can review the following PR.
- https://github.com/pynecone-io/pynecone/pull/1094

Two PR are  okay for me, because they at least give one way for the developer who want to push pure source code on git.

Reply:
@ElijahAhianyo  Can you give one reason about why git should keep `__pycache__` ? 
Because if you don't approve this PR and don't give reason. 
@chrisrosner  will keep to reject this PR 
- https://github.com/pynecone-io/pynecone/pull/1094

You keep to reject this PR, and chrisrosner keep to reject another PR ... 
Then  it must be a disaster for all pynecone app developer.